repository/SmalltalkCI-Core.package/SCIMetacelloLoadSpec.class/instance/password.st

```diff
@@ -0,0 +1,4 @@
+accessing
+password
+
+	^ password ifNil: [ ^ '' ]
```

Comment:
Why return an empty string if nil? Wouldn't it make all checks easier if it would return `nil` when the password is not set?

Reply:
This implementation was copied from `SCIMonticelloLoadSpec` where you fell back to `''` as well. :-) But yes, I would prefer to return `nil`, too, I will change that.

Reply:
(I don't know whether Monticello or Metacello depend on a non-nil string, but to prevent this, we do not need to pollute the public accessor protocol of SCI.)

Reply:
Good catch, maybe that should be cleaned up at some point. It looks like `MCHttpRepository` can handle `nil` values for `#password` and `#user`. If you like, you can clean this up as part of this PR.

Reply:
I could do this, however, #486 touches exactly the same methods, so I'd rather refactor the nil values in that PR ...

Reply:
Right!